.github/actions/gradle-setup/action.yml

```diff
@@ -62,3 +69,16 @@ runs:
     - name: Move Registration Service packages out of build directory
       run: mv RegistrationService ${{ runner.temp }}
       shell: bash
+
+    # Install Identity Hub packages into ~/.m2.
+    # This action only runs if the packages could not be restored from the cache.
+    - name: Build Identity Hub packages
+      run: |
+        ./gradlew publishToMavenLocal -Pskip.signing
```

Comment:
that will become obsolete once [this PR](https://github.com/eclipse-dataspaceconnector/RegistrationService/pull/14) is merged.

Reply:
@paullatzelsperger switch to the latest Identity Hub and removed skip.signing. But still seems to fail: bc13cf0cb8589b792eef733c7cf7b3422476add5bc13cf0cb8589b792eef733c7cf7b3422476add5

Anyways for publishing to the local Maven repository it should not play that much of a role. So suggesting to leave it in.